.circleci/config.yml

```diff
@@ -272,12 +268,11 @@ jobs:
     description: "Run integration tests for Flutter"
     resource_class: macos.x86.medium.gen2
     macos:
-      xcode: 13.4.1
+      xcode: 14.3.1
     shell: /bin/bash --login -o pipefail
     steps:
       - checkout
-      - setup-flutter:
-          version: '3.3.10'
+      - setup-flutter
```

Comment:
Since this wont' work yet, I would revert the chages in `macos-integration-test`

Reply:
Switched to flutter 3.3.10 which was the last stable version before OpenGL support was deprecated. Looks like tests pass now :)